sql/moz-fx-data-shared-prod/app_store_syndicate/dataset_metadata.yaml

```
@@ -0,0 +1,12 @@
+friendly_name: app_store_syndicate
+description: |
+  Syndicated App Store data from Fivetran. User-facing views into this data can be found
+  in the ads dataset
+dataset_base_acl: syndicate
+user_facing: false
+syndication:
+  prod:
+    syndicated_project: "moz-fx-data-bq-fivetran"
+    syndicated_dataset: "app_store"
+  administer_views: false
```

I've updated the docs on https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27920974/BigQuery+shared-prod#BigQuery(shared-prod)-Addingsyndicatedatasets. Fivetran is one of the cases where this value should be set to true (because DSRE automation has direct management access to that infra, unlike e.g. [fakespot infra](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/fakespot_syndicate/dataset_metadata.yaml) managed by other teams). There's an example of that in https://github.com/mozilla/bigquery-etl/commit/7761dc2c6057aacf825951351aba38892663ff80 which I've also linked in the doc.

```suggestion
  administer_views: true
```